{"name": "test_successful_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'AuthPage' object has no attribute 'equal'", "trace": "auth_page = <pom.index_page.AuthPage object at 0x000002A31A079810>\n\n    @allure.label(\"owner\", \"Sergey!!\")\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.description('Checking successful login')\n    @pytest.mark.ui\n    def test_successful_login(auth_page):\n        with allure.step('input correct username'):\n            auth_page.enter_correct_username()\n        with allure.step('input correct password'):\n            auth_page.enter_correct_password()\n        with allure.step('click login'):\n            auth_page.click_login_button()\n        with allure.step('check successful login'):\n>           auth_page.successful_login()\n\ntests\\test_auth.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pom.index_page.AuthPage object at 0x000002A31A079810>\n\n    def successful_login(self):\n        expected_text = 'PRODUCTSS'\n        actual_text = self.to_text('css', Locators.TITLE)\n>       self.equal(expected_text, actual_text)\nE       AttributeError: 'AuthPage' object has no attribute 'equal'\n\npom\\index_page.py:34: AttributeError"}, "description": "Checking successful login", "steps": [{"name": "input correct username", "status": "passed", "start": 1666205289330, "stop": 1666205289710}, {"name": "input correct password", "status": "passed", "start": 1666205289710, "stop": 1666205289909}, {"name": "click login", "status": "passed", "start": 1666205289909, "stop": 1666205290183}, {"name": "check successful login", "status": "broken", "statusDetails": {"message": "AttributeError: 'AuthPage' object has no attribute 'equal'\n", "trace": "  File \"C:\\Users\\SPlaksin\\PycharmProjects\\AutomationTesting2\\tests\\test_auth.py\", line 28, in test_successful_login\n    auth_page.successful_login()\n  File \"C:\\Users\\SPlaksin\\PycharmProjects\\AutomationTesting2\\pom\\index_page.py\", line 34, in successful_login\n    self.equal(expected_text, actual_text)\n"}, "start": 1666205290183, "stop": 1666205290361}], "attachments": [{"name": "log", "source": "464e6a4c-59df-4d63-a29f-2fed726498cd-attachment.txt", "type": "text/plain"}], "start": 1666205289330, "stop": 1666205290361, "uuid": "dd6b6589-d9a6-4484-b169-cff96e19e0cc", "historyId": "5112c4f747a2a8839bb3b2b704d00471", "testCaseId": "90fe1fd32e71fe1bbc81fd489be5da6d", "fullName": "tests.test_auth#test_successful_login", "labels": [{"name": "owner", "value": "Sergey!!"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "ui"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "NB-100211"}, {"name": "thread", "value": "12156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}